Lib/threading.py

```diff
@@ -800,8 +799,20 @@ class is implemented.
         assert group is None, "group argument must be None for now"
         if kwargs is None:
             kwargs = {}
+        if name:
+            name = str(name)
+        if not name:
```

Comment:
```suggestion
        else:
```

Reply:
I would like to avoid empty thread name, and so I check that the result of str(name) is non-empty. Do you think that it's not worth it?

Reply:
I agree with Serhiy's suggestion, if someone uses an empty string it was probably intentional. I don't think it's worth trying to catch it.